game-core/src/main/java/games/strategy/triplea/settings/ClientSetting.java

```diff
@@ -163,36 +170,23 @@
    * </p>
    */
   public static void initialize() {
-    setPreferences(Preferences.userNodeForPackage(ClientSetting.class));
-  }
-
-  @VisibleForTesting
-  static void setPreferences(final Preferences preferences) {
```

Comment:
this method was moved lower. Since it is visible for testing, we can do parameter null check at point of caller, so the  precondition check was moved to 'initialize'